data_summaries/CollectiveCognition.json

```diff
@@ -0,0 +1,47 @@
+{
+    "CollectiveCognition": {
+        "Unique Dataset Identifier": "CollectiveCognition",
+        "Dataset Name": "Collective Cognition Chat Data",
+        "Paper Title": "",
+        "Dataset URL": "https://huggingface.co/datasets/CollectiveCognition/chats-data-2023-10-16",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/CollectiveCognition/chats-data-2023-10-16",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": 0,
```

Comment:
empty string

Reply:
There is no associated arxiv URL or Papers with code for this data

Reply:
Sorry, as in if there is no CorpusID then put "", not 0